structure/linkedlist/doubly.go

```diff
@@ -91,6 +91,45 @@ func (ll *Doubly[T]) DelAtEnd() (T, bool) {
 	return retval, true
 }
 
+// DelInMiddle deletes the node in middle of linked list and returns its value.
+// Returns false if list is empty
+func (ll *Doubly[T]) DelInMiddle() (T, bool) {
+	return ll.DelByPos(ll.Count() / 2)
+}
```

Comment:
My point was not to edit the old function but rather completely remove it since the caller can easily delete the middle one by themselves.

Reply:
Ok will do that

Reply:
removed the code as you have requested